lib/presentation/util/helper/responsive_widget.dart

```diff
@@ -0,0 +1,78 @@
+// LinShare is an open source filesharing software, part of the LinPKI software
+// suite, developed by Linagora.
+//
+// Copyright (C) 2020 LINAGORA
+//
+// This program is free software: you can redistribute it and/or modify it under the
+// terms of the GNU Affero General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later version,
+// provided you comply with the Additional Terms applicable for LinShare software by
+// Linagora pursuant to Section 7 of the GNU Affero General Public License,
+// subsections (b), (c), and (e), pursuant to which you must notably (i) retain the
+// display in the interface of the “LinShare™” trademark/logo, the "Libre & Free" mention,
+// the words “You are using the Free and Open Source version of LinShare™, powered by
+// Linagora © 2009–2020. Contribute to Linshare R&D by subscribing to an Enterprise
+// offer!”. You must also retain the latter notice in all asynchronous messages such as
+// e-mails sent with the Program, (ii) retain all hypertext links between LinShare and
+// http://www.linshare.org, between linagora.com and Linagora, and (iii) refrain from
+// infringing Linagora intellectual property rights over its trademarks and commercial
+// brands. Other Additional Terms apply, see
+// <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf>
+// for more details.
+// This program is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+// more details.
+// You should have received a copy of the GNU Affero General Public License and its
+// applicable Additional Terms for LinShare along with this program. If not, see
+// <http://www.gnu.org/licenses/> for the GNU Affero General Public License version
+//  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
+//  the Additional Terms applicable to LinShare software.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+class ResponsiveWidget extends StatelessWidget {
+  final Widget largeScreen;
+  final Widget mediumScreen;
+  final Widget smallScreen;
+
+  static final double widthDevice = window.physicalSize.width;
+
+  static final int minLargeWidth = 2048;
+  static final int minMediumWidth = 1536;
```

Comment:
IMO, this pixel is not realistic for mobile and tablet device, please explain about unit using for width and height. I want to make clear about `logicalPixel` and `physicalPixel`

Reply:
Base on https://material.io/design/layout/pixel-density.html#pixel-density.

I understand that, your width is in `physicalPixel`. It will be different in other device with other density. So I think, we should not depend on `physicalPixel`. We need to depend on `logicalPixel`. Please investigate more. Thanks

Reply:
Note: we can access logical pixel via: `MediaQuery.of(context).size`

Reply:
After talking with designer, it seems I misunderstood the device size. You're right, using logicalPixel is way more effective and sustainable. I put breakpoints in description, this is working fine for me tell me what you think :)